apps/client/src/widgets/community/components/filter-dropdown/filter-dropdown.tsx

```diff
@@ -24,7 +26,7 @@ const FilterDropDown = ({
       <div className={styles.DropDownTitle}>
         {optionTitle}
         <div
-          className={`${styles.DropDownIcon} ${styles.isRotate({ isRotate: isIconRotate && !open })}`}
+          className={`${styles.DropDownIcon} ${styles.isRotate({ isRotate: isIconRotate && !open })} ${iconBackground === 'whiteBackground' ? styles.iconWhiteBackground : styles.iconTransparent}`}
```

Comment:
```suggestion
          className={styles.DropDownIcon({
            background: iconBackground,
            isRotate: isIconRotate && !open,
          })}
```

이렇게 처리하는 건 어떠신가요? 
이렇게 하고 dropDownIcon 는 recipe 로 variants 에 rotated : true / false + background : white / transparent 로 분기해두는 게 조금 더 가독성이 좋을 것 같아요.

Reply:
의견 감사합니다 ㅎㅎ 반영해두겠습니다